src/decision/test/lidar-decision-test.cpp

```diff
@@ -6,10 +6,65 @@
 
 #include <LidarDecision.h>
 #include <gtest/gtest.h>
+#include <vector>
+
+TEST(LidarDecision, obstacle_in_range){
```

Comment:
You'll probably need more then one test for `obstacle_in_range`. I would test with obstacles the angle range but not within the maximum distance, obstacles not within the maximum distance but within the minimum angle range, obstacles just outside of the angle range but within the maximum distance, etc.